rfc/text/advanced/ap_31_authentication.md

```diff
@@ -1,4 +1,4 @@
-# Authentication Methods {#auth-methods}
+## Authentication {#authorization}
```

Comment:
I think this should use the same name and anchor for correctness :)
```suggestion
## Authentication {#authentication}
```

Reply:
Discuss this later, let's discuss how the full structure first.